lib/reality/extras/economics/indicator.rb

```diff
@@ -0,0 +1,53 @@
+require 'quandl'
+Quandl::ApiConfig.api_version = '2015-04-09'
+
+module Reality
+  module Extras
+    module Economics
+      class Indicator
+        attr_reader :name, :code, :unit, :scale, :country
+
+        def initialize(country, code, info)
+          Quandl::ApiConfig.api_key = Reality.config.fetch('keys', 'quandl')
+
+          @name = info['name']
+          @unit = info['unit']
+          @scale = info['scale'] || 1
+          @code = code
+          @country = country
+        end
+
+        def current
+          return 'no data' if data.nil?
+          history.last.value
+        end
+
+        def history
+          return 'no data' if data.nil?
+          data.values.select { |v| v['date'] <= Date.today }.sort_by { |d| d['date'] }
+        end
+
+        def prediction
+          return 'no data' if data.nil?
+          data.values.select { |v| v['date'] > Date.today }.sort_by { |d| d['date'] }
+        end
+
+        def data
+          @data ||= fetch
+        end
+
+        def inspect
+          "#<%s (%s)>" % ['Economics::Indicator', name]
```

Comment:
In fact, it is not very good to "imitate" classnames which are not real (this form of inspect implies `Economic::Indicator` is a complete classname)